src/service_client_lib.h

```diff
@@ -89,6 +89,12 @@ class ServiceClient : public GrpcClient<chirp::ServiceLayer::Stub> {
   ReturnCodes SendMonitorRequest(const std::string &username,
                                  std::vector<Chirp> *const chirps);
 
+  // Send a stream request to the server with the tag `tag`
+  // returns OK if this operation succeeds
+  // returns other error codes if this operation fails
+  ReturnCodes SendStreamRequest(const std::string &tag,
+                                 std::vector<Chirp> *const chirps);
```

Comment:
This signature is a bit funny; why a const pointer of vector?

Reply:
Fixed

Reply:
Because the function shouldn't modify the content of the pointted vector. It is issued from another function.

Reply:
Because the function modifies the content of the monitor but cannot point the vector to another one.